README.md

```diff
@@ -97,6 +100,7 @@ The configuration defines the various components of the project you want to mana
 | healthChecks          | The various checks to run to ensure the container is healthy (see below for list) |             |
 | terminationSignals    | Signals which should be sent to a specific container when it should be shut down  |             |
 | terminationChecks     | The various checks to run to ensure that the container is ready to be shut down   | connections |
```

Comment:
I wanted to explicitly reference containers by name, since our data containers don't need to be managed by gantry.  We could, alternatively, set them up as their own gantry components and this could be a list of component names from which to mount volumes?  I wasn't quite sure about naming conventions here, either.  Let me know what you think.